script/script.js

```diff
@@ -0,0 +1,147 @@
+const toolbar = {
+    view: "toolbar",
+    id: "toolbar",
+    paddingX: 12,
+    css: "webix_dark",
+    elements: [{
+            view: "label",
+            label: "My App",
+        },
+        {},
+        {
+            view: "button",
+            type: "icon",
+            icon: "wxi-user",
+            label: "Profile",
+            width: 90,
+            css: "webix_transparent"
+        }
+    ]
+};
+const menuList = {
+    view: "list",
+    id: "list",
+    template: "#title#",
+    scroll: false,
+    css: "webix_layout_clean",
+    data: menuListSet
+};
+const menuLink = {
+    template: "<span class='webix_icon wxi-check'></span> Connected",
+    height: 30,
+    type: "clean",
+    css: "span-green"
+};
+const menu = {
+    maxWidth: 200,
+    minWidth: 100,
+    type: "clean",
+    rows: [
+        menuList,
+        {},
+        menuLink
+    ],
+    css: "menu"
+
+};
+const datatable = {
+    view: "datatable",
+    id: "datatable",
+    autoConfig: true,
```

Comment:
either use autoConfig (for auto column generation), or set column explicitely
there's no need to use both techniques simultaneously